### PR TITLE
CPUID.HWMonitor version 1.49

### PR DESCRIPTION
**Pull request by TimoS254**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95825)

### CODE REVIEW

File: manifests/c/CPUID/HWMonitor/1.49/CPUID.HWMonitor.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://download.cpuid.com/hwmonitor/hwmonitor_1.49.exe
+  InstallerSha256: 33BF467C7A9A89DE617A71AD8749660AE7DFB89E10A5A6198B6CA0EF475F47BD
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://download.cpuid.com/hwmonitor/hwmonitor_1.49.exe
+  InstallerSha256: 33BF467C7A9A89DE617A71AD8749660AE7DFB89E10A5A6198B6CA0EF475F47BD
+ManifestType: installer
+ManifestVersion: 1.3.0
```
Comment (denelon) [inline manifests/c/CPUID/HWMonitor/1.49/CPUID.HWMonitor.installer.yaml:24]:
```suggestion
ManifestVersion: 1.4.0

```

File: manifests/c/CPUID/HWMonitor/1.49/CPUID.HWMonitor.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- monitor
+- hardware
+- temerature-monitoring
+- sensor
+- fan-speed
+- thermal
+- hard-drive
+- temps
+ManifestType: defaultLocale
+ManifestVersion: 1.3.0
```
Comment (denelon) [inline manifests/c/CPUID/HWMonitor/1.49/CPUID.HWMonitor.locale.en-US.yaml:34]:
```suggestion
ManifestVersion: 1.4.0

```

File: manifests/c/CPUID/HWMonitor/1.49/CPUID.HWMonitor.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.3.0.schema.json
+
+PackageIdentifier: CPUID.HWMonitor
+PackageVersion: 1.49
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.3.0
```
Comment (denelon) [inline manifests/c/CPUID/HWMonitor/1.49/CPUID.HWMonitor.yaml:8]:
```suggestion
ManifestVersion: 1.4.0

```

**Review by denelon — CHANGES_REQUESTED** [review]

**Comment by denelon** [comment]
There is also a more recent version of wingetcreate.

**Comment by denelon** [comment]
https://github.com/microsoft/winget-create/releases/tag/v1.2.5.0

**Review by Trenly — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/95825#discussion_r1100564950

**Comment by stephengillie** [comment]
Hi @Timo5254,

The package installs, and launches without error. But it doesn't remain running, doesn't enter a usable state, and closes soon after - in either VM nor sandbox. It also doesn't appear to log to its own folder nor the Windows Event Logs. 

Is there a dependency? Alternatively, is there a way to log? I looked at the developer's website and only found troubleshooting steps for the Pro version.

**Comment by ItzLevvie** [comment]
Close with reason: A newer version is available;

**Pull request closed without merging**